.appveyor.yml

```diff
@@ -119,5 +114,5 @@ for:
       # Runs only in Linux
       - "LAMBDA_BUILDERS_DEV=1 pytest -vv tests/integration"
 
-      # Validate Code was formatted with Black
-      - "/tmp/black --check setup.py tests aws_lambda_builders"
+      # Validate Code was formatted with Black, black is only supported in Python 3
+      - if [[ $PYTHON_VERSION != "2"* ]]; then black --check setup.py tests aws_lambda_builders; fi
```

Comment:
is there any reason that you use `$PYTHON_VERSION != "2"*` but not `$PYTHON_VERSION == "3"*` or `$PYTHON_VERSION -gt "2"`?

Reply:
Changed to `if [[ $PYTHON_VERSION > "3" ]]; then black --check setup.py tests aws_lambda_builders; fi`

Reply:
This might be pedantry, but the `>` operator [compares lexicographically](https://www.gnu.org/software/bash/manual/html_node/Bash-Conditional-Expressions.html); is that the intention?

Reply:
either lexicographically or numerically should be able to specify "python 3 or higher" right?